manage_breast_screening/templates/record_a_mammogram/appointment_cannot_go_ahead.jinja

```diff
@@ -1,3 +1,76 @@
 {% extends "wizard_step.jinja" %}
+{% from 'components/checkboxes/macro.jinja' import checkboxes %}
+{% from 'components/input/macro.jinja' import input %}
+{% from 'components/radios/macro.jinja' import radios %}
 
-{% set title = "Appointment cannot go ahead" %}
\ No newline at end of file
+{% set title = "Appointment cannot go ahead" %}
+{% set caption = participant.full_name %}
+
+{% block form %}
+{% macro conditionalCheckboxInput(params) %}
+  {% set details_field_name = params.field_name + "_details" %}
+  {% set input_params = {
+    'label': {
```

Comment:
[Code style] I think we're going to converge on double quotes rather than single quotes in the templates (@colinrotherham is migrating the Nunjucks template to this style)

We haven't set up linting yet though, so can come back to it later if you prefer.